src/cart/cart.scss

```
@@ -0,0 +1,172 @@
+body {
+    justify-content: center;
+}
+
+.cart {
+    width: 1136px;
+    margin: 40px auto 80px;
+}
+
+.cart-nav {
+    width: 66px;
+    height: 16px;
+    display: flex; /* Zmiana na display: flex */
+}
+
+.cart-nav-iconBox {
+    width: 16px;
+    height: 16px;
+    justify-content: center;
+    display: flex; /* Zmiana na display: flex */
+}
+
+.cart-nav-icon {
+    height: 10px;
+    width: 6px;
+    margin: auto;
+}
+
+.cart-nav-link {
+    text-decoration: none;
+    line-height: 15px;
+    font-weight: 700;
+    font-size: 12px;
+    color: #89939A;
+    margin-left: 4px;
+    margin-right: 15px;
+    margin-top: 1px;
+}
+
+.cart-title-text {
+    width: 100%;
+    line-height: 56px;
+    font-family: 800;
+    font-size: 48px;
+    margin-top: 16px;
+    margin-bottom: 32px;
+}
+
+.cart-contentBox {
+    display: flex;
+}
+
+.cart-productCardsBox {
+    width: 752px;
+    margin-right: 16px;
+    // border-radius: 16px;
+    // border: 1px solid    #E2E6E9;
+}
+
+.cart-checkoutBox {
+    width: 368px;
+    height: 206px;
+    border-radius: 16px;
+    border: 1px solid    #B4BDC3;
+}
+
+.cart-sumBox {
+    width: 321px;
+    height: 87px;
+    border-bottom: solid 1px #E2E6E9;
+    margin: 23px;
+}
+
+.cart-checkout-button {
+    width: 320px;
+    height: 48px;
+    margin: 24px;
+    margin-top: 0;
+    border-radius: 8px;
+    border: #216CFF;
+    background-color: #216CFF;
+    font-size: 14px;
+    line-height: 21px;
+    color: #FFFFFF;
+    font-weight: 700;
+}
+
+.cart-message {
+    margin-top: 20px;
+    width: 100%;
+    height: 60px;
+    border-radius: 8px;
+    border: #27AE60;
+    background-color: #27AE60;
+    font-size: 14px;
+    line-height: 21px;
+    color: #FFFFFF;
+    font-weight: 700;
+    display: none;
+}
+
+.message-active {
+    display: block;
+}
+
+
+@media screen and (min-width: 641px) and (max-width: 1199px) {
```

Breakpoint should be set to 650
```suggestion
@media screen and (min-width: 650px) and (max-width: 1199px) {
```